test/integrated-devnet-tests/with-stderr-to-STDERR/check.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+source ../scripts/check-devnet-is-not-running.sh
+
+check_devnet_is_not_running
+
+EXPECTED_STDOT="Account #0"
```

Comment:
STDOT -> STDOUT (here and in other files)

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/179/commits/cd6433e6ee81fc0b5b82b549552b4fdfb46506b2